src/Controller/EditCustomerOptionsAction.php

```diff
@@ -57,7 +57,6 @@ public function __invoke(Request $request): Response
 
         if ($orderItemForm->isSubmitted() && $orderItemForm->isValid()) {
             $this->orderItemOptionUpdater->updateOrderItemOptions($orderItem, $orderItemForm->getData());
-            $orderItem->recalculateUnitsTotal();
```

Comment:
Why is this not needed anymore?

Reply:
We don't change the unit totals. So no need to recalculate them.